kili/helpers.py

```diff
@@ -382,9 +385,19 @@ def get_file_paths_to_upload(files: Tuple[str, ...]) -> List[str]:
             file_paths.extend(
                 [sub_item for sub_item in glob.glob(item) if os.path.isfile(sub_item)]
             )
-
-    file_paths.sort()
-    return file_paths
+    file_paths_to_upload = [path for path in file_paths if file_check_function(path)]
+    if len(file_paths_to_upload) == 0:
+        raise ValueError(
+            "No files to upload. Check that the paths exist and that the file type is correct"
+        )
+    if verbose:
+        for path in file_paths:
+            if path not in file_paths_to_upload:
+                print(f"{path:30} SKIPPED")
+        if len(file_paths_to_upload) != len(file_paths):
+            print("Paths skipped either do not exist or point towards an incorrect file")
+    file_paths_to_upload.sort()
+    return file_paths_to_upload
 
 
 def file_check_function_from_input_type(input_type: str):
```

Comment:
I removed all this part in my PR, you miss a rebase ?

Reply:
I have put it back since it was needed.